apps/condo/domains/common/utils/testSchema/index.js

```diff
@@ -65,6 +65,10 @@ export const expectToThrowAccessDeniedErrorToObj = async (testFunc) => {
 }
 
 /**
+ * @deprecated
+ * THIS FUNCTION IS DEPRECATED. We want to to separate access rights errors from non-authorization errors!
+ * Please use expectToThrowAccessDeniedErrorToObjects!
+ *
```

Comment:
Function `expectToThrowAccessDeniedErrorToObjects` should not be deprecated, because it tests use cases, when access rights of **authenticated** user agents are restricted.